packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/openshift/validateOpenshiftURL.ts

```diff
@@ -1,12 +1,20 @@
 import { validateURL, ValidationMsg } from '../../common';
 
 export const validateOpenshiftURL = (url: string | number): ValidationMsg => {
+  // For a newly opened form where the field is not set yet, set the validation type to default.
+  if (url === undefined) {
+    return {
+      type: 'default',
+      msg: 'The URL of the Openshift Virtualization API endpoint, for example: https://example.com:6443 .',
+    };
+  }
+
   // Sanity check
   if (typeof url !== 'string') {
     return { type: 'error', msg: 'URL is not a string' };
   }
 
-  const trimmedUrl: string = url.toString().trim();
+  const trimmedUrl: string = url?.toString().trim();
```

Comment:
url can't be undefined here, see line 5, so we don't need the `?` it can't be a string so we can drop the `toString` too